test/e2e/kubelet_perf.go

```diff
@@ -242,7 +246,10 @@ var _ = Describe("Kubelet [Serial] [Slow]", func() {
 			itArg := testArg
 			podsPerNode := itArg.podsPerNode
 			name := fmt.Sprintf(
-				"for %d pods per node over %v", podsPerNode, monitoringTime)
+				"resource tracking for %d pods per node", podsPerNode)
+			if podsPerNode >= 35 {
```

Comment:
We run "[Feature:Performance]" in continuous scalability test. I don't think we want to run this in that suite. Maybe "[Feature:NodePerformance]"